### ISSUE TITLE
Thelia thelia:update PDOException

### ISSUE THREAD

**Issue by sean3z** [thelia/thelia#1032]
Running `php Thelia thelia:update` produces the following error (then discontinues?)
```bash
user@box [~/public_htm/]# php Thelia thelia:update

Welcome to the update database process
Please wait ...

  [PDOException]
  SQLSTATE[42S21]: Column already exists: 1060 Duplicate column name 'cart_id'

thelia:update
```

**Comment by lunika** [comment]
Hi,
From which version did you start the update ? 

Thanks

**Comment by sean3z** [comment]
I received the error when updating from `2.0.3` to `2.0.4`

**Comment by lunika** [comment]
as you can see [here](https://github.com/thelia/thelia/blob/2.0.3/setup/thelia.sql#L659) the cart_id column doesn't exist in order table for the version 2.0.3 of Thelia.

Did you use an intermediate version between Thelia 2.0.3 and 2.0.4 ?

**Comment by sean3z** [comment]
Not that I'm aware of. Thelia `2.0.3` was installed via Softaculous (by someone else) and I've upgraded using the `2.0` [UPDATE.md](https://github.com/thelia/thelia/blob/master/UPDATE.md)

**Comment by pierr0t** [comment]
I reported the same problem in forums. I had a 2.0.4 installed as 2.0.4 (never updated), but if you look in the "config" table, the "thelia_version" shows 2.0.3 ...
So the update process tries to update to 2.0.4 although it's already a 2.0.4, hence the error. I solved the problem by manually (phpmyadmin) changing that value for 2.0.4, and then update went flawlessly.
And I checked another install, same ... so I guess there is an error in the install script for 2.0.4, it populates the config table with a wrong 2.0.3 value.
